### PR TITLE
Update The Readme.md File so it makes a better connection with the FAQ

### PR DESCRIPTION
**Pull request by sajjaadf**
As the Title Says

### CODE REVIEW

**Comment by JL2210** [comment]
Nice. I always wanted "Compatability", because "Compatibility" is useless. :)
Could you change that?

File: README.md [hunk 1]
```diff
@@ -59,6 +59,13 @@ On a rooted Google OnHub, the command needs to be run with the "chronos" user. I
     # su - chronos
     Password:
     $ curl -Ls git.io/vddgY | bash
+    
+Compatablity
+------------
+
+If you I Crouton installed, Do I get support too?
+
+We will try to help, but we may not be able to. We will not give any extreme effort to provide support for Crouton (e.g. Removing or modifying packages in Chromebrew)
```
Comment (uberhacker) [inline README.md]: This should go in the FAQ and not README.md.

**Comment by sajjaadf** [comment]
sure

File: README.md (hunk 1 above)
Comment (sajjaadf) [reply README.md]: It is in the FAQ

**Comment by JL2210** [comment]
Actually, come to think about it, this might be unneeded. As long as people are installing both and they work, then our job is done.

**Comment by sajjaadf** [comment]
Cant we leave it there?

**Comment by uberhacker** [comment]
This should go in the FAQ, not README.md.  If you want to bring more attention to the FAQ section, you might say something like _Please check out the [Wiki](https://github.com/skycocker/chromebrew/wiki) to find out more information about Chromebrew including helpful tips, resource links and frequently asked questions_.

**Pull request closed without merging**